common/metrics/defs.go

```diff
@@ -951,6 +991,27 @@ var MetricDefs = map[ServiceIdx]map[int]metricDefinition{
 		CacheMissCounter:                             {metricName: "cache-miss", metricType: Counter},
 		AcquireLockFailedCounter:                     {metricName: "acquire-lock-failed", metricType: Counter},
 		WorkflowContextCleared:                       {metricName: "workflow-context-cleared", metricType: Counter},
+		HistorySize:                                  {metricName: "history-size", metricType: Timer},
```

Comment:
i think the counter metrics type are all named *Counter, so should be the timers?